client/cypress/e2e/MinimapTest.cy.ts

```diff
@@ -44,7 +44,7 @@ testEachZone((zone: Cypress.PrismZone) => {
     it(`Testing: user changes x coordinates input in the form, the targeted mininode position changes correctly`, () => {
       if (zone.adminUser) {
         cy.wait(getReqAlias).then(() => {
-          const randX = Math.floor(Math.random() * 9) * 10 + 10;
+          const randX = 50;
```

Comment:
Why is this changed when we're getting a random value for X?

Reply:
fixed, passed locally, but with many trials